mod_ci/controllers.py

```diff
@@ -545,8 +545,7 @@ def start_ci():
         elif event == "pull_request":
             # If it's a PR, run the tests
             commit = ''
-            gh_commit = None
-            if payload['action'] == 'opened':
+            if payload['action'] == 'opened' or payload['action'] == 'synchronize':
```

Comment:
I'd rather see a payload['action'] in ['opened', 'synchronize'], that's easier to extend afterwards.